pyproject.toml

```diff
@@ -1,46 +1,49 @@
 [build-system]
-  requires = [
-    "setuptools",
-    "setuptools_scm",
-    "wheel"
-  ]
-  build-backend = 'setuptools.build_meta'
-
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
 
 [project]
-  name = "astropy-iers-data"
-  dynamic = ["version"]
-  description = "IERS Earth Rotation and Leap Second tables for the astropy core package"
-  readme = "README.rst"
-  requires-python = ">=3.8"
-  license = {file = "LICENSE.rst"}
-  keywords = ["astropy", "data", "IERS"]
-  authors = [
-    {name = "Astropy Developers", email = "astropy.team@gmail.com"}
-  ]
-  classifiers = [
-    "Intended Audience :: Science/Research",
-    "Intended Audience :: Developers",
-    "License :: OSI Approved :: BSD License",
-    "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3",
-  ]
+name = "astropy-iers-data"
+dynamic = ["version"]
+description = "IERS Earth Rotation and Leap Second tables for the astropy core package"
+readme = "README.rst"
+requires-python = ">=3.8"
+license = {file = "LICENSE.rst"}
```

Comment:
How licenses should be expressed has been changed in Python and we might as well update that now. 
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license-and-license-files

Reply:
Good catch, fixed in [`ca71c34` (#39)](https://github.com/astropy/astropy-iers-data/pull/39/commits/ca71c3466cff34cae6825287068f60eb81887819)